website/blog/2022/04/17/api-observability.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "API Observability with Apache APISIX Plugins"
+keywords:
+- Apache APISIX
+- API Gateway
+- Observability
+description: In this blog post, we can leverage the power of some [Apache APISIX](https://apisix.apache.org/) Observability Plugins and take a look at how to set up these plugins, how to use them to understand API behavior, and later solve problems that impact our users.
+tags: [Technology,Observability]
```

Comment:
May need one or two authors when you find one.
```suggestion
authors:
  - name: ""
    title: "Author"
    url: ""
    image_url: ""
description: In this blog post, we can leverage the power of some [Apache APISIX](https://apisix.apache.org/) Observability Plugins and take a look at how to set up these plugins, how to use them to understand API behavior, and later solve problems that impact our users.
tags: [Technology,Observability]
```

Reply:
![image](https://user-images.githubusercontent.com/99874763/164957130-b9973fd5-87bf-4181-8ba5-5435880d0a2f.png)

Is this ok？

Reply:
Yes if you click you the `Boburmizro` and it goes to his github page.